contracts/InterestRateModels/InterestRateModel.sol

```diff
@@ -31,4 +31,13 @@ contract InterestRateModel {
         uint reserves,
         uint reserveFactorMantissa
     ) external view returns (uint);
+
+    /**
+     * @notice Calculates the utilization rate for the market
+     * @param cash The total amount of cash the market has
+     * @param borrows The total amount of borrows the market has outstanding
+     * @param reserves The total amount of reserves the market has
+     * @return The utilization rate (as a percentage)
+     */
+    function utilizationRate(uint cash, uint borrows, uint reserves) public pure returns (uint);
```

Comment:
I suppose that in another PR we'll move this `utilizationRate` from here to VToken, as we did in IL to allow the rebalancing.